.eslintrc

```diff
@@ -3,9 +3,60 @@
   "parser": "babel-eslint",
   "plugins": ["lodash"],
   "rules": {
-    "lodash/prefer-lodash-method": "error"
-  },
-  "settings": {
-    "lodash": 3
+    "arrow-parens": 0,
```

Comment:
This is a whole lot of rule exceptions. Did these come in as a prescripted set from somewhere or mostly preferential one-offs?

Reply:
When I upgraded to the latest versions of the various eslint packages we had been using, it started flagging a tonne of new issues.  

I decided that it would be better to simply add rule exceptions for most of the new issues, rather than fix all occurrences to satisfy the new lint rules, which would have amounted to a massive amount of changes to the codebase.

I do think that we should likely tackle this at some point, but I feel like we should perhaps figure out exactly what rules/configs we want to hold to, and likely consolidate `viz` and `mobile` to use the same configs so we have some consistency moving forward.

What do you think, Chris?

Reply:
That seems reasonable. I do think it simplifies a lot of things when we're able to use these sorts of "standards" (eg airbnb eslint config) more out of the box than making a lot of edits, but I do agree that applying the fixes directly to this PR would have been foolhardy. I think that we should look at a consistent eslint across our codebases as much as possible (`viz`, `mobile`, `uploader` and `blip` should all be reasonable to keep together I think), and that it should probably happen in a concerted effort.